kotsadm/kustomize/overlays/dev/deployment.yaml

```diff
@@ -56,6 +56,36 @@ spec:
               value: "1"
             # - name: DISABLE_OUTBOUND_CONNECTIONS  
             #   value: "true"
+      initContainers:
+        - name: init-dex-db
+          command:
+          - "psql"
+          args:
+          - "-h"
+          - "kotsadm-postgres"
+          - "-U"
+          - "kotsadm"
+          - "-c"
+          - "CREATE DATABASE dex;"
+          - "-c"
+          - "CREATE USER dex;"
+          - "-c"
+          - "ALTER USER dex WITH PASSWORD '$(DEX_PGPASSWORD)';"
+          - "-c"
+          - "GRANT ALL PRIVILEGES ON DATABASE dex TO dex;"
+          env:
+          - name: PGPASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: password
+                name: kotsadm-postgres
+          - name: DEX_PGPASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: password
+                name: kotsadm-dex-postgres
+          image: kotsadm
```

Comment:
this actually works?

Reply:
that's how it is in base, and we don't patch that for the main container